Extensions/vanilla_audio.js

```diff
@@ -26,13 +26,37 @@ XKit.extensions.vanilla_audio = new Object({
 		}
 	},
 
-	run: function() {
+	run: async function() {
 		"use strict";
 		this.running = true;
+
+		await XKit.css_map.getCssMap();
+
+		if (XKit.page.react) {
```

Comment:
trying to get the CSS map on non-react pages isn't necessary

```suggestion
		if (XKit.page.react) {
			await XKit.css_map.getCssMap();
```

Reply:
Fixed!